drivers/video/ov5640.c

```diff
@@ -573,12 +659,236 @@ static int ov5640_stream_stop(const struct device *dev)
 	return ov5640_write_reg(&cfg->i2c, SYS_CTRL0_REG, SYS_CTRL0_SW_PWDN);
 }
 
+#define TEST_PATTERN_ENABLE  BIT(7)
+#define TEST_PATTERN_ROLLING BIT(6)
+#define TEST_PATTERN_BAR     (0 << 0)
+#define TEST_PATTERN_SQUARE  (2 << 0)
+
+static const uint8_t test_pattern_val[] = {
+	0,
+	TEST_PATTERN_ENABLE | TEST_PATTERN_BAR | (1 << 2),
+	TEST_PATTERN_ENABLE | TEST_PATTERN_BAR | (1 << 2) | TEST_PATTERN_ROLLING,
```

Comment:
Just for reference for whoever curious about this `(1 << 2)`:
![2024-09-16-113728_1331x166_scrot](https://github.com/user-attachments/assets/2bcda934-3f2c-4132-b35c-6701db68b595)

If interested:

```c
#define TEST_PATTERN_NOGRADIENT (0 << 2)
#define TEST_PATTERN_VGRADIENT1 (1 << 2)
#define TEST_PATTERN_HGRADIENT  (2 << 2)
#define TEST_PATTERN_VGRADIENT2 (3 << 2)
```